src/redux/actions/books.js

```diff
@@ -0,0 +1,45 @@
+// constants
+export const ADD_BOOK = 'ADD_BOOK';
+export const REMOVE_BOOK = 'REMOVE_BOOK';
+export const TOGGLE_BOOK = 'TOGGLE_BOOK';
```

Comment:
Please ensure that the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`. according to the project requirement.

Reply:
done